### ISSUE TITLE
[MODEL EVALUATION REQUEST] Qwen/Qwen2-72B

### ISSUE THREAD

**Issue by saattrupdan** [ScandEval/ScandEval#459]
### Model ID

Qwen/Qwen2-72B

### Model type

Decoder model (e.g., GPT)

### Model languages

- [X] Danish
- [X] Swedish
- [X] Norwegian (Bokmål or Nynorsk)
- [X] Icelandic
- [X] Faroese
- [X] German
- [X] Dutch
- [X] English

### Merged model

Not a merged model

**Comment by KasperGroesLudvigsen** [comment]
Is this still relevant now that Qwen/Qwen2.5-72B-Instruct has been included?

**Comment by saattrupdan** [comment]
> Is this still relevant now that Qwen/Qwen2.5-72B-Instruct has been included?

Good point, I'll remove it - anyone can suggest reopening if there is an interest 🙂
